### PR TITLE
Adding service status coverage to backup and restore

### PR DESCRIPTION
**Pull request by Griffin-Sullivan**
Coverage for https://bugzilla.redhat.com/show_bug.cgi?id=1978764. The backup and restore tests are what we run with remote DB so this should automatically cover the scenario in the BZ.

### CODE REVIEW

**Comment by Griffin-Sullivan** [comment]
trigger: test-robottelo
pytest: tests/foreman/maintain/test_backup_restore.py::test_positive_backup_restore

**Comment by Griffin-Sullivan** [comment]
trigger: test-robottelo
pytest: tests/foreman/maintain/test_backup_restore.py::test_positive_backup_restore tests/foreman/maintain/test_backup_restore.py::test_positive_puppet_backup_restore

**Review by pondrejk — APPROVED** [review]

File: tests/foreman/maintain/test_backup_restore.py [hunk 1]
```diff
@@ -448,7 +448,7 @@ def test_positive_puppet_backup_restore(
     result = sat_maintain.cli.Backup.run_backup(
         backup_dir=subdir,
         backup_type=backup_type,
-        options={'assumeyes': True, 'plaintext': True, 'skip-pulp-content': False},
+        options={'assumeyes': True, 'plaintext': True, 'skip-pulp-content': False, 't': '50M'},
```
Comment (shubhamsg199) [inline tests/foreman/maintain/test_backup_restore.py:451]: For better readability 
```suggestion
        options={'assumeyes': True, 'plaintext': True, 'skip-pulp-content': False, 'split-pulp-tar': '50M'},
```

**Comment by Griffin-Sullivan** [comment]
Capsule backup and restore blew up for some reason (will need to look into that separately), but the satellite `test_positive_backup_restore` all passed so the changes for that test are good. I need to make an update to the changes under `test_positive_puppet_backup_restore` and then we can rerun those to see if it works.

**Comment by Griffin-Sullivan** [comment]
trigger: test-robottelo
pytest: tests/foreman/maintain/test_backup_restore.py::test_positive_puppet_backup_restore

File: tests/foreman/maintain/test_backup_restore.py [hunk 2]
```diff
@@ -448,7 +448,12 @@ def test_positive_puppet_backup_restore(
     result = sat_maintain.cli.Backup.run_backup(
         backup_dir=subdir,
         backup_type=backup_type,
-        options={'assumeyes': True, 'plaintext': True, 'skip-pulp-content': False},
+        options={
+            'assumeyes': True,
+            'plaintext': True,
+            'skip-pulp-content': False,
+            'split-pulp-tar': '50M',
+        },
```
Comment (jameerpathan111) [inline tests/foreman/maintain/test_backup_restore.py:456]: I see that `split-pulp-tar` option is already covered in `test_positive_backup_split_pulp_tar` and `test_positive_backup_all` tests. So we don't really need to cover it here again, we can just add BZ number to the existing test cases.

Comment (Griffin-Sullivan) [reply tests/foreman/maintain/test_backup_restore.py:456]: Oh nice I didn't think to check again after switching the name!

**Review by jameerpathan111 — APPROVED** [review]

**Pull request merged**